src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/ScenarioRunner.cs

```diff
@@ -38,6 +39,7 @@ internal static class ScenarioRunner
             toolRunner.DiagnosticPortPath = diagnosticPortPath;
             toolRunner.DisableAuthentication = true;
             toolRunner.DisableHttpEgress = disableHttpEgress;
+            toolRunner.CustomStartupHookPath = customStartupHookPath;
```

Comment:
Why does the `ToolRunner` have a setting for specifying the startup hook? We shouldn't be applying startup hooks to .NET Monitor.

Reply:
Yep - I was having a lot of issues initially getting these tests to run end-to-end and was experimenting to figure out the issue, but never went back and pulled out some of the unnecessary stuff like this - it should be removed now.